sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_client_manager.py

```diff
@@ -323,28 +334,85 @@ def __init__(
                 )
             )
             self._setup_failover_endpoints()
+            if load_balancing_enabled:
+                random.shuffle(self._replica_clients)
             return
         raise ValueError("Please pass either endpoint and credential, or a connection string with a value.")
 
+    def get_next_client(self) -> Optional[_ConfigurationClientWrapper]:
+        """
+        Get the next client to be used for the request. find_active_clients needs be called before this method or None
```

Comment:
That is a difficult dependency to maintain. If `get_next_client` needs `find_active_clients` to be invoked first, we should do it in `get_next_client`.